homeassistant/components/switcher_kis/switch.py

```diff
@@ -0,0 +1,179 @@
+"""Home Assistant Switcher Component Switch platform."""
+
+from logging import getLogger
+from typing import Any, Callable, cast, Dict, Optional
+
+from homeassistant.components.switch import ATTR_CURRENT_POWER_W, SwitchDevice
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.exceptions import PlatformNotReady
+
+from . import (
+    ATTR_AUTO_OFF_SET, ATTR_DEVICE_NAME, ATTR_ELECTRIC_CURRNET,
+    ATTR_REMAINING_TIME, DATA_DEVICE, DOMAIN, SIGNAL_SWITCHER_DEVICE_UPDATE)
+
+_LOGGER = getLogger(__name__)
+
+DEPENDENCIES = ['switcher_kis']
```

Comment:
No more need for this line since we switched to use manifest.json generate dependencies.

Reply:
Fixed in this [commit](https://github.com/home-assistant/home-assistant/pull/22325/commits/26542f2c2973ee111aeb14b7ccd5f3dadaca65b5).